guides/common/modules/con_hammer-configuration.adoc

```diff
@@ -21,11 +20,11 @@ $ hammer -d --version
 
 [NOTE]
 ====
-Loading configuration for many CLI modules can slow down the execution of `hammer` commands.
+Loading configuration for many CLI modules can slow down the execution of Hammer commands.
 In such a case, consider disabling CLI modules that are not regularly used.
 ====
-Apart from saving credentials as described in xref:hammer-authentication[], you can set several other options in the *~/.hammer/* configuration directory.
-For example, you can change the default log level and set log rotation with the following directives in *~/.hammer/cli_config.yml*.
+Apart from saving credentials as described in xref:hammer-authentication[], you can set several other options in the `~/.hammer/` configuration directory.
+For example, you can change the default log level and set log rotation with the following directives in `~/.hammer/cli_config.yml`.
```

Comment:
This file was not present on my Foreman+Katello Server (3.16/4.18). Did the path change or is it not present by default?

I only found YAML files in `~/.hammer/cli.modules.d/`.

I would be OK investigating after this PR.